src/WebJobs.Script/Rpc/LanguageWorkerChannel.cs

```diff
@@ -265,7 +272,7 @@ internal void PublishWorkerProcessReadyEvent(FunctionEnvironmentReloadResponse r
             _eventManager.Publish(wpEvent);
         }
 
-        internal void PublishRpcChannelReadyEvent(RpcEvent initEvent)
+        internal void SendDownloadDependencyRequestOrPublishRpcChannelReadyEvent(RpcEvent initEvent)
```

Comment:
This steps needs to be decoupled. Otherwise, enabling placeholders will not work.
PublishRpcChannelReadyEvent is the first step.
SendDownloadDependencyRequest should be next step if ManagedDependency feature is enabled.